grammar.js

```diff
@@ -783,6 +783,8 @@ module.exports = grammar({
       '*',
     ),
 
+    parameter: $ => "?",
```

Comment:
The `$1` syntax is also quite common and should be straightforward to capture here if you're so inclined.